comps/llms/src/text-generation/README.md

```diff
@@ -188,6 +190,7 @@ curl http://${host_ip}:${TEXTGEN_PORT}/v1/chat/completions \
 [meta-llama/Meta-Llama-3-8B-Instruct]: https://huggingface.co/meta-llama/Meta-Llama-3-8B-Instruct
 [meta-llama/Meta-Llama-3-70B-Instruct]: https://huggingface.co/meta-llama/Meta-Llama-3-70B-Instruct
 [Phi-3]: https://huggingface.co/collections/microsoft/phi-3-6626e15e9585a200d2d761e3
+[falcon-3.1-74b]: https://huggingface.co
```

Comment:
Why no direct link to model?

Reply:
The model link is not public yet, will add after it's public